gamemodes/terrortown/gamemode/client/cl_main.lua

```diff
@@ -421,6 +421,9 @@ local function RoundStateChange(o, n)
 
 		-- clear decals in cache from previous round
 		util.ClearDecals()
+
+		-- resets bone positions that fixes broken fingers on bad addons
+		modelbuilder.ResetBonePositions(LocalPlayer():GetViewModel())
```

Comment:
Seeing this, i really dislike to have this on the modelbuilder module. Especially since we now use this function in a context different than the weapon base.

Reply:
hm, I still think it fits nicely in that library - especially after we rename it